Spokestack/TextToSpeech.swift

```diff
@@ -85,13 +90,86 @@ import CryptoKit
         self.synthesize(input: input, success: play)
     }
     
+    @available(iOS 13.0, *)
+    public func synthesizePublisher(_ input: TextToSpeechInput) -> AnyPublisher<TextToSpeechResult, Error> {
```

Comment:
Am I correct in thinking that `synthesizePublisher` is the single-input function, and `TextToSpeechQueue. synthesize` is the mutli-input function?

Reply:
Correct